README-MULTISITE.md

```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
 Finally, The variable `rgw_zone` is set to "default" to enable compression for clusters configured without rgw multi-site.
 If multisite is configured `rgw_zone` should not be set to "default".
 
-For more defail information on multisite please visit: https://docs.ceph.com/docs/master/radosgw/multisite/
+For more defail information on multisite please visit: <https://docs.ceph.com/docs/master/radosgw/multisite/>.
 
-# Deployment Scenario #1: Single Realm & Zonegroup with Multiple Ceph Clusters
+## Deployment Scenario #1: Single Realm & Zonegroup with Multiple Ceph Clusters
 
-## Requirements
+### Requirements
 
 * At least 2 Ceph clusters
 * 1 RGW per cluster
 * Jewel or newer
 
-## Configuring the Master Zone in the Primary Cluster
+### Configuring the Master Zone in the Primary Cluster
 
 This will setup a realm, master zonegroup and master zone in the Ceph cluster.
 Since there is only 1 realm, 1 zonegroup, and 1 zone for all the rgw hosts, only `group_vars/all.yml` needs to be edited for mulitsite conifguration.
 If there is one more that one rgw being deployed in this configuration, the rgw(s) will be added to the master zone.
 
 1. Generate System Access and System Secret Keys
 
-```
-echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys.txt
-echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys.txt
-```
+    ```bash
+    echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys.txt
+    echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys.txt
+    ```
+
 2. Edit `group_vars/all.yml` for the 1st cluster
 
-```
-rgw_multisite: true
+    ```yaml
+    rgw_multisite: true
 
-rgw_zone: juneau
-rgw_zonegroup: alaska
-rgw_realm: usa
+    rgw_zone: juneau
+    rgw_zonegroup: alaska
+    rgw_realm: usa
 
-rgw_zonemaster: true
-rgw_zonesecondary: false
+    rgw_zonemaster: true
+    rgw_zonesecondary: false
 
-rgw_zonegroupmaster: true
+    rgw_zonegroupmaster: true
 
-rgw_multisite_proto: http
-rgw_zone_user: edward.lewis
-rgw_zone_user_display_name: "Edward Lewis"
+    rgw_multisite_proto: http
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
 
-system_access_key: 6kWkikvapSnHyE22P7nO
-system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
-```
-**Note:** `rgw_zonemaster` should have the value of `true` and `rgw_zonesecondary` should be `false`. Both values always need to be defined when running multisite.
+    system_access_key: 6kWkikvapSnHyE22P7nO
+    system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
+    ```
```

Comment:
Any reason for adding extra space characters there ?

Reply:
Yes, that's for the ordered list items to be part of the same list.

For instance:

1. This is the first item:

```
Some code
```

2. This is not the second item, it's a new ordered list that starts at 2.

But this works:

1. This is the first item:

    ```
    Some code
    ```

2. This is the second item.

(You can inspect the HTML to see the difference.)

Reply:
This was maybe clearer before I squashed the commits, this was originally added in f24e2078c77438c6d4b6c2bb1f7855b0eb71c1f7.